src/app_service/common/utils.nim

```diff
@@ -1,10 +1,13 @@
-import json, random, times, strutils, os, re, chronicles
+import json, random, times, strutils, sugar, os, re, chronicles
 import nimcrypto
 import signing_phrases
 
 const STATUS_DOMAIN* = ".stateofus.eth"
 const ETH_DOMAIN* = ".eth"
 
+proc arrayContains*[T](arr: seq[T], value: T): bool = 
+  return arr.filter(x => x == value).len > 0
```

Comment:
You could use [any](https://nim-lang.org/docs/sequtils.html#any%2CopenArray%5BT%5D%2Cproc%28T%29) from sequtils, most of the time will be more performant.

Reply:
I overlooked it, will change, thanks